jwst/cube_build/cube_build_io_util.py

```diff
@@ -234,6 +231,19 @@ def read_cubepars(par_filename,
                     instrument_info.SetHighEMSMTable(table_wave, table_sroi,
                                                      table_wroi, table_scalerad)
 
+            elif weighting == 'drizzle':
+                # TODO when we get a new cubepars reference file with wavelength
```

Comment:
Oh, so the wavelength range is in fact being read from the cubepars ref file, but just from a different section/extension of the ref file. So the description needs to be updated to make it more explicit exactly what's being changed and how.